src/graphql/event/http.rs

```diff
@@ -114,12 +114,14 @@ impl HttpThreat {
         &self.inner.user_agent
     }
 
-    async fn request_len(&self) -> usize {
-        self.inner.request_len
+    /// The length of the request in string within the range representable by a `usize`
```

Comment:
it is recommended to modify the document to end with a "."

Reply:
Done